src/v2/view-builder/views/signin/SignInWithIdps.js

```diff
@@ -19,7 +22,8 @@ export default View.extend({
     const jsonData = View.prototype.getTemplateData.apply(this, arguments);
 
     return Object.assign(jsonData, {
-      addSeparateLine: this.options.addSeparateLine,
+      addSeparateLineAtTop: this.options.addSeparateLine && !this.options.addSeparateLineAtBottom,
```

Comment:
instead of this flag `this.options.addSeparateLine` we can just use idpButtons value ? For eg.. if buttons exists then we have to add the divider..

Reply:
we can simplify this too..  If it's not `isPrimaryIdpDisplay` then obviously it's default.  We don't need these two variables.

jsonData.isPrimaryIdpDisplay = this.option. isPrimaryIdpDisplay
return jsonData;

Reply:
sounds good, I'll simplify the arguments here